python/ray/data/tests/test_dataset_text.py

```diff
@@ -19,6 +19,10 @@
 from ray.tests.conftest import *  # noqa
 
 
+def _to_lines(rows):
```

Comment:
shall we also add a unit test here if input text file is empty? or all lines are empty lines?

Reply:
done.